### PR TITLE
responsive screenshot example

### PR DESCRIPTION
**Pull request by salihsagdilek**
Captures the full height document even if it's not showing on the screen or captures with the provided range of screen sizes.
A basic example for taking a screen shot using phantomjs

examples >
      phantomjs responsive-screenshot.js https://nodejs-dersleri.github.io/
      phantomjs responsive-screenshot.js https://nodejs-dersleri.github.io/ pdf
      phantomjs responsive-screenshot.js https://nodejs-dersleri.github.io/ true
      phantomjs responsive-screenshot.js https://nodejs-dersleri.github.io/ png true

2015-11-22 12-39-22 mingw64 d app rssg
rssg

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,3 +60,52 @@ bin/
 *.class
 build/
 .gradle/
+
```
Comment (sodabrew) [inline .gitignore]: These seem to be gitignore entries better suited for a personal `~/.gitignore` - do they need to be in the phantomjs repo?

**Comment by sodabrew** [comment]
Nice example for full-screen screenshots!

File: .gitignore (hunk 1 above)
Comment (salihsagdilek) [inline .gitignore]: yea, its my mistake, i will remove gitignore file and will make new pull request.

**Comment by Vitallium** [comment]
@salihsagdilek no need to make a new pull request, just push your changes and the pull request will be updated automatically.

**Comment by salihsagdilek** [comment]
@Vitallium ok, i fixed it.

**Comment by Vitallium** [comment]
Landed! Thank you! https://github.com/ariya/phantomjs/commit/f7ad5e1649f38abec82eb64521f630aba02eed9d

**Pull request closed without merging**